.pipelines/ci.yml

```diff
@@ -17,8 +17,52 @@ jobs:
     inputs:
       targetType: 'inline'
       script: 'bazelisk build -c dbg //src/cc/lib:wrapper --config=windows'
+  - task: PowerShell@2
+    inputs:
+      targetType: 'inline'
+      script: 'Set-ItemProperty -Path $(Build.SourcesDirectory)\bazel-bin\src\cc\lib\wrapper.dll -Name "IsReadOnly" -Value $false'
+
+  - task: EsrpCodeSigning@4
+    inputs:
+      ConnectedServiceName: 'gnn-sign'
+      FolderPath: '$(Build.SourcesDirectory)\bazel-bin\src\cc\lib'
+      Pattern: '*.dll'
+      signConfigType: 'inlineSignParams'
+      SessionTimeout: '90'
+      MaxConcurrency: '50'
+      MaxRetryAttempts: '5'
+      PendingAnalysisWaitTimeoutMinutes: '5'
+      inlineOperation: |
+        [
+            {
+                "KeyCode" : "CP-230012",
```

Comment:
Should this be a secret?

Reply:
no, this is just a certificate type, there are multiple.